### ISSUE TITLE
Does not work with dry-system 0.18.0

### ISSUE THREAD

**Issue by jandudulski** [dry-rb/dry-rails#37]
App fails with:

```ruby
NoMethodError:
  private method `boot_files' called for #<Dry::System::Booter:0x0000000004e35068>
  Did you mean?  booted
```

**Comment by danderozier** [comment]
I was literally about to post the same bug. Thanks @jandudulski!

**Comment by timriley** [comment]
I'll take a look at this shortly, thanks for the reports.

**Comment by timriley** [comment]
Thanks for the quick report of this issue, folks!

Here's how I've addressed it:

- Released 0.2.1 of dry-rails which depends on dry-system 0.17.x specifically (i.e. it won't permit an upgrade to dry-system 0.18.0) - [see changes](https://github.com/dry-rb/dry-rails/compare/v0.2.0...v0.2.1)
- Released 0.18.1 of dry-system, which makes the `Booter#boot_files` method public again (the source of your particular issue here) - [see changes](https://github.com/dry-rb/dry-system/compare/v0.18.0...v0.18.1)
- Released 0.3.0 of dry-rails, which depends on dry-system 0.18.1 or higher, including another tweak to the container setup to properly configure dry-system 0.18.0's new `bootable_dirs` setting - [see changes](https://github.com/dry-rb/dry-rails/compare/v0.2.1...v0.3.0)

So your options here are to:

- Upgrade to 0.2.1 of dry-rails to keep things working in the short term
- Upgrade to 0.3.0 to permit future dry-system upgrades

Both of these should work just fine, I think :)

Can you please have a go and let me know how it works out?

Sorry again for the unexpected breakages!

/cc @solnic

**Comment by jandudulski** [comment]
Works, thanks :heart:
